docs/docs/Components/components-vector-stores.md

```diff
@@ -659,3 +659,36 @@ For more information, see the [Weaviate Documentation](https://weaviate.io/devel
 | Name           | Type       | Description                |
 |----------------|------------|----------------------------|
 | search_results | List[Data] | Results of similarity search |
+
+
+## GridGain Vector Store
+
+This component implements a Vector Store using GridGain with CSV ingestion and search capabilities.
+
+### Inputs
+
+| Name              | Type          | Description                                                                   |
+|-------------------|---------------|----------------------------------------------------------------               |
+| cache_name        | String        | The name of the cache within GridGain where vectors will be stored (required) |
```

Comment:
Lets make required fields and defaults consistent with our chat memory component, for example:
The name of the cache within GridGain where vectors will be stored. Required.
Its slightly inconsistent with other fields, but they do not provide this info at all.